docs/api/1.x/parser.md

```diff
@@ -7,6 +7,8 @@ Whenever an error occurs during the VAST parsing, the parser will call on his ow
 
 * VAST error `101`: VAST schema validation error.
 
+* VAST error `102`: VAST version of response not supported.
```

Comment:
```suggestion
* VAST error `102`: VAST response version not supported.
```

Reply:
Actually I edited the wrong file, this documentation is for old versions 1.x. I moved it to the correct one. The error description comes from IAB https://wiki.iabtechlab.com/index.php?title=VAST_Error_Code_Troubleshooting_Matrix 
![Uploading image.png…]()